client/site-profiler/components/loading-screen/index.tsx

```diff
@@ -35,16 +35,16 @@ export const LoadingScreen = () => {
 			if ( progress >= 100 ) {
 				setProgress( 0 );
```

Comment:
Why do we set it back to `0`? I don't think that's needed anymore as we don't want to get it reset.

Reply:
You're right, I updated the condition here in 136fd95758